### ISSUE TITLE
Refactor USB Source constructor to generate a unique/incremental device-location for each source.

### ISSUE THREAD

**Issue by rjhowell44** [prominenceai/deepstream-services-library#855]
The current implementation does not set the device-location, leaving the setting to default to `/dev/video0`.  . Therefore, only once USB camera can be used. The constructor will be updated to generate the unique device-location. Rules are as follows.

A unique  device-location is assigned to each USB Source on creation, starting with `/dev/video0`, followed by `/dev/video1`, and so on. The default assignment   can be overridden by calling dsl_source_usb_device_location_set. The call will fail if the given device-location is not unique. If a source is deleted,  the device-location will be re-assigned to a new USB Source if one is created.

**Comment by rjhowell44** [comment]
Merged into v0.26.alpha
